# Pan/Tilt/Zoom limits for the simulated camera
# -179 to 179
min_pan: -3.14159
max_pan: 3.12413936106985
# -90 to 90
min_tilt: -1.5707963267948966
max_tilt: 1.5707963267948966
# 5x digital zoom applied to the simulated camera stream
min_logical_zoom: 1
max_logical_zoom: 5

# Hardware zoom limits are ignored by the action server node
# These parameters exist only to keep consistency with the Axis and Flir action server nodes
min_zoom: 1
max_zoom: 5
